documentation/src/main/asciidoc/reference/indexing-plan.asciidoc

```diff
@@ -106,8 +106,6 @@ See <<mapper-orm-indexing-manual-indexingplan-process-execute>> for more informa
 [[indexing-plan-synchronization]]
 == [[mapper-orm-indexing-automatic-synchronization]] Synchronization with the indexes
 
-include::components/mapper-orm-only-note.asciidoc[]
-
```

Comment:
Good thinking, but then there's a lot more to be rewritten in this section to make it applicable to both the Standalone mapper and the ORM mapper.

I'd have a hard time listing everything, so it would probably be easier if I give it a try. I'll leave this comment here, unresolved, so that I remember to have a look before we merge.

Reply:
> there are still docs to be updated and probably some tests in the docs module.

yeah.. once I've realized that methods would be called differently and that there might be even different names for the beans, I've decided to leave the docs unchanged for now and only mark an intent to modify the section so that it's "reusable" for both mappers 🙈

Reply:
Just pinging on this one to see if you'd want to add anything in the docs or leave it for some other RP?

Reply:
I think what you did is probably enough for now, we'll have another look when we address https://hibernate.atlassian.net/browse/HSEARCH-4616